pmm-app/src/pmm-qan/panel/components/Details/Explain/Explain.tools.ts

```diff
@@ -28,7 +28,7 @@ export const processClassicExplain = (classic): ClassicExplainInterface => {
     .map((e) => (String(e) ? e.trim() : ''))
     .filter(Boolean)
     .reduce((acc, row, index) => {
-      acc[headerList[index].accessor] = row;
+      acc[headerList[index].accessor] = row.split(',').join(', ');
```

Comment:
How about replaceAll method?
```
acc[headerList[index].accessor] = row.replaceAll(',', ', ');
```

Reply:
The same result but it seems to be faster.
Thanks.